### PR TITLE
Adds build wrapper script

### PR DESCRIPTION
**Pull request by conorsch**
Using an Ansible playbook to read in the YAML metadata for each container and kick off a build. Builds are tagged with timestamp of year, month, and day in UTC, e.g. `20170925`. Right now the playbook does *not* push images automatically, figured we should take it for a spin first.

Is it madness to use a Makefile to call Ansible to build Docker containers? Probably it is. Should we just use `docker-compose` to build and push these images? Probably we should. 

To test, run `make build` locally and confirm you get new images.

### CODE REVIEW

**Comment by msheiny** [comment]
>Is it silly to use Ansible and make to build docker images? Yes.
>Should we be using docker-compose instead? Almost certainly.

What does docker-compose do that ansible doesnt do? If anything it appears that we lose functionality by using docker-compose. With ansible's docker's modules we can easily spin up multiple docker instance and link them together in a similar fashion that docker-compose does.

**Comment by msheiny** [comment]
also im going to drop this here :) if for some reason we need to utilize whatever features from compose we are looking to gain from a switch --> http://docs.ansible.com/ansible/latest/docker_service_module.html

tl;dr  -- im really trying to fight introducing docker-compose to our workflow vs. utilizing ansible

**Review by msheiny — APPROVED** [review]
I think this is a really solid base -- I have a few concerns long-term that I made tickets for see #4 , and #3

I'm not going to block though since I have other changes to get in and I dont want this to be lost.
